Tests/test_PDB_parse_pdb_header.py

```diff
@@ -0,0 +1,50 @@
+"""Unit tests for PARTS of the parse_pdb_header module of Bio.PDB"""
+
+import unittest
+import warnings
+
+try:
+    import numpy
+    from numpy import dot  # Missing on old PyPy's micronumpy
```

Comment:
I used test_PDB_MMCIFParser.py as a template for my test-file. I'll update the imports.